script.js

```diff
@@ -47,4 +47,42 @@ form.addEventListener('submit', (e) => {
     email.style.border = '3px solid red';
     error.textContent = 'X   Email should be in lowerCase';
   }
+});
+
+//  Local  storage---------------->
+const inputEmail = document.getElementById('form-email');
+const inputName = document.getElementById('fullname');
+const inputMessage = document.getElementById('text-area');
+
+// eslint-disable-next-line no-unused-vars
+function formData() {
```

Comment:
- [ ] I kindly suggest that you ensure that when the users fills in the form without submitting, the entered value should be saved in the browser. Kindly see the video below for a demo:

https://www.loom.com/share/9bd7f216a9aa4c35b5db117e88e66227

You can achieve this by having an input event listener on your form that saves data to localStorage.

## EXAMPLE
```
const contactForm= {
  name: obj ? obj.name : '',
  email: obj ? obj.email : '',
  message: obj ? obj.message : '',
};

form.addEventListener('input', (e) => {
  const targetElement = e.target;

  if (targetElement.getAttribute('type') === 'text') {
    contactForm.name = targetElement.value;
  } else if (targetElement.getAttribute('type') === 'email') {
    contactForm.email = targetElement.value;
  } else {
    contactForm.message = e.target.value;
  }

  localStorage.setItem('formData', JSON.stringify(contactForm));
});

```